src/applications/mhv/medications/util/helpers.js

```diff
@@ -48,3 +49,20 @@ export const validateField = (fieldValue, date = false) => {
   }
   return 'None noted';
 };
+
+/**
+ * @param {String} fieldValue value that is being validated
+ */
+export const getImageUri = fieldValue => {
+  const folderNames = ['1', '2', '3', '4', '5', '6', '7', '8', '9'];
+  let folderName = fieldValue
+    ? fieldValue.replace(/^0+(?!$)/, '').substring(0, 1)
+    : '';
+  const fileName = `NDC${fieldValue}.jpg`;
+  // check if the foldername is one of the listed folders. if not, set folder name to “other”
+  if (!folderNames.includes(folderName)) {
```

Comment:
would it be simpler/more readable to do something like this? (untested pseudocode)

`if (!folderName.match(/^[1-9]$/))`

Reply:
it is from original java code of krish.
![Screen Shot 2023-09-11 at 9 47 51 AM](https://github.com/department-of-veterans-affairs/vets-website/assets/132001256/5082dc1c-f0aa-4cda-b78b-3e0ecfaf1713)